src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
@anthonylangsworth please keep the change, please file the issue in this repo, assign to me and point to your fix. I'll take care of making sure this gets ported to full .NET. I think the test should still be expecting prefixes in the elements though - was this also like that on netfx?